specs/vep-1243-vdk-meta-jobs/README.md

```diff
@@ -90,6 +110,20 @@ In this context, a component is any separate software process.
 
 -->
 
+![high-level-diagram.png](high-level-diagram.png)
+
+The vdk-meta-jobs plugin is a VDK component that enables the orchestration of Data Jobs.
```

Comment:
nit: Is `vdk-meta-jobs plugin` clearly mentioned or defined before? If yes, you've already set up context, and may refer to it like  the plugin. I would actually expect to have this information provided until the Summary is wrapped up.

My current understanding is, that the product of VEP-1243 is `vdk-meta-jobs` plugin, so the high-level design has to do with the plugin, and the detailed design has to do with that plugin as well

Reply:
It is mentioned first in the Goals section so I cut down all the rest references to "the plugin".

Reply:
Looking at clear explanation at the beginning of the document, of what implementation is specified by this VEP-1243.
Once stated, then referring to it may be simplified